### PR TITLE
Sample Census Page

### PR DESCRIPTION
**Pull request by ianHunterClipd**
First pass at Sample Census page. Page needs to be modified to use FormattedMessages for i18n and SASS instead of css in js.

### CODE REVIEW

File: client/components/SampleCensus.js [hunk 1]
```diff
@@ -2,330 +2,159 @@ import React from 'react';
...
-                          <SampleCensusCheckbox
-                            checked
-                            readOnly
-                          />
-                        )
-                      }}
-                    />
-                  </span>
-                </header>
+    //REMOVE ME
```
Comment (JMStudiosJoe) [inline client/components/SampleCensus.js:106]: ? still need to remove it?

**Review by JMStudiosJoe — CHANGES_REQUESTED** [review]

**Comment by JMStudiosJoe** [comment]
Just a question about a comment and a for loop

File: client/components/SampleCensus.js (hunk 1 above)
Comment (ianHunterClipd) [reply client/components/SampleCensus.js:106]: At the moment this is creating the list of 9 placeholder questions to be rendered, so we still need it if we want to have a demonstrable page on Heroku. I am trying to figure out a more concise way to dynamically create FormattedMessage components within the CensusQuestionCard while still getting their default text translated.

Comment (JMStudiosJoe) [reply client/components/SampleCensus.js:106]: +1 k will merge for now

**Review by JMStudiosJoe — APPROVED** [review]

**Pull request merged**